label_studio_ml/examples/watsonx_llm/data_transfer_app.py

```diff
@@ -0,0 +1,163 @@
+import csv
+import logging
+import os
+import prestodb
+import zipfile
+from flask import Flask, request, jsonify, Response
+from label_studio_sdk.client import LabelStudio
+from typing import List
+
+logger = logging.getLogger(__name__)
+_server = Flask(__name__)
+
+
+def init_app():
+    return _server
+
+
+# @_server.route('/')
+# # ‘/’ URL is bound with hello_world() function.
+# def hello_world():
+#     return 'Hello World'
+
+
+@_server.route('/health', methods=['GET'])
+@_server.route('/', methods=['GET'])
+def health():
+    return jsonify({
+        'status': 'UP',
+        'model_class': MODEL_CLASS.__name__
+    })
+
+
+@_server.route('/upload', methods=['POST'])
+def upload_to_watsonx():
+
+    # First, collect data from the request object passed by label studio
+    input_request = request.json
+    action = input_request["action"]
+    annotation = input_request["annotation"]
+    task = input_request["task"]
+
+    # Connect to Label Studio
+    client = connect_ls()
+    data = get_data(annotation, task, client)
+
+    # Then, connect to WatsonX.data via prestodb
```

Comment:
does it work with default "None" values? if not, defaults should be provided explicitly

Reply:
Can't set defaults, as these values are unique to each user, but set to throw an exception if they are not provided.